src/renderer/views/TechnologyPicker/TechnologiesPicker.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import StyledSwitch from '../../components/Switch/Switch';
+import './TechnologiesPicker.scss';
+
+const TechnologiesPicker = () => {
+  const { t, i18n } = useTranslation();
+  let [useSCSS, changeUseSCSS] = useState(true);
+  let [useTP, changeTP] = useState(true);
```

Comment:
We are not going to be using the `useState` in this component but for reference the definition should be `const` instead of `let`.

Instead of `useState` we're using redux and the reducer prepared in #144. So in order to get the information about the state of the sass and typescript toggles it is enough to use the `newPublication` selector and destructuring the `useSass` and `useTypescript` properties.

```ts
  const { useSass, useTypescript } = useSelector(newPublication);
```

These can then be passed to the switch as the `checked` props.